crates/bevy_ui/src/ui_node.rs

```diff
@@ -171,10 +231,13 @@ impl Default for Direction {
     }
 }
 
+/// Whether to use Flexbox layout
 #[derive(Copy, Clone, PartialEq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect_value(PartialEq, Serialize, Deserialize)]
 pub enum Display {
+    /// Use flexbox
     Flex,
+    /// Use no layout
```

Comment:
This affects both layout and rendering, the node and it's children won't be displayed